sdcm/tester.py

```diff
@@ -141,6 +141,43 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+class silence:  # pylint: disable=invalid-name
```

Comment:
Can we add verbosity to the silence, so if `verbose=True`
it will log with info log level as follows:

```
Silently running 'tag_ami_with_result'...
Finished 'tag_ami_with_result'. No errors were silenced.
```
When exception happened, log with warning log level:
```
Silently running 'tag_ami_with_result'...
Finished 'tag_ami_with_result'. ValueError exception was silenced.
```

If `self.name` is not `None` use it as a name instead name of the function/method.

Reply:
Done